packages/fluentui/react-northstar/src/components/Attachment/Attachment.tsx

```diff
@@ -168,13 +155,18 @@ Attachment.propTypes = {
   actionable: PropTypes.bool,
   description: customPropTypes.itemShorthand,
   header: customPropTypes.itemShorthand,
-  icon: customPropTypes.shorthandAllowingChildren,
+  icon: customPropTypes.itemShorthand,
```

Comment:
Why was this changed? Does this means that we are changing the API here? I would expect that even if we have different component, to keep the same API..

Reply:
Merge issue, should be reverted